### ISSUE TITLE
HQDeringmod.avsi missing dither.avsi dependency

### ISSUE THREAD

**Issue by Selur** [realfinder/AVS-Stuff#145]
Just wanted to let you know that since you removed dither.avsi, HQDeringmod.avsi doesn't work any more since it relies on dither.avsi.

**Comment by realfinder** [comment]
you can use the original one from http://avisynth.nl/index.php/Dither_tools dither-x.xx.x.zip

**Comment by Selur** [comment]
Yeah, I'm aware of that, just thought it should be mentioned since it seems the only script now relaying on external scripts.
